features/test/src/test.ts

```diff
@@ -5,360 +5,386 @@ let quads: Array<RDF.Quad> = [];
-    if (result.length !== 1)
-        throw Error(
-            `Expected array with 1 element, got ${result.length} elements`
-        );
-    if (!result[0].equals(expectedResult))
-        throw Error(
-            `Expected element equal to '${JSON.stringify(
-                expectedResult
-            )}', got '${JSON.stringify(result[0])}'`
-        );
-}
-
-export async function canGetArrayWithSingleQuadWithBlankNodeObjectFromPolyInMatch(): Promise<void> {
-    console.log(
-        "canGetArrayWithSingleQuadWithBlankNodeObjectFromPolyInMatch()"
-    );
-    const expectedResult = QuadBuilder.fromQuad(quads[0])
-        .withObject(pod.dataFactory.blankNode(quads[0].object.value))
-        .build();
-    const result = await polyIn.match({});
-    if (result.length !== 1)
-        throw Error(
-            `Expected array with 1 element, got ${result.length} elements`
-        );
-    if (!result[0].equals(expectedResult))
-        throw Error(
-            `Expected element equal to '${JSON.stringify(
-                expectedResult
-            )}', got '${JSON.stringify(result[0])}'`
-        );
-}
-
-export async function canGetArrayWithSingleQuadWithLiteralObjectFromPolyInMatch(): Promise<void> {
-    console.log("canGetArrayWithSingleQuadWithLiteralObjectFromPolyInMatch()");
-    const expectedResult = QuadBuilder.fromQuad(quads[0])
-        .withObject(pod.dataFactory.literal(quads[0].object.value))
-        .build();
-    const result = await polyIn.match({});
-    if (result.length !== 1)
-        throw Error(
-            `Expected array with 1 element, got ${result.length} elements`
-        );
-    if (!result[0].equals(expectedResult))
-        throw Error(
-            `Expected element equal to '${JSON.stringify(
-                expectedResult
-            )}', got '${JSON.stringify(result[0])}'`
-        );
-}
-
-export async function canGetArrayWithSingleQuadWithNamedNodeGraphFromPolyInMatch(): Promise<void> {
-    console.log("canGetArrayWithSingleQuadWithNamedNodeGraphFromPolyInMatch()");
-    const expectedResult = QuadBuilder.fromQuad(quads[0])
-        .withGraph(pod.dataFactory.namedNode(quads[0].graph.value))
-        .build();
-    const result = await polyIn.match({});
-    if (result.length !== 1)
-        throw Error(
-            `Expected array with 1 element, got ${result.length} elements`
-        );
-    if (!result[0].equals(expectedResult))
-        throw Error(
-            `Expected element equal to '${JSON.stringify(
-                expectedResult
-            )}', got '${JSON.stringify(result[0])}'`
+// TODO: Workaround for missing console.debug on iOS
+console.debug = console.debug || console.log;
+
+const tests = {
+    /*
+      Tests prefixed with _ are disabled, and will not be exposed.  Most of
+      those are disabled because they test functionality we would like to have
+      one day, but don't actually need (and thus, support) yet.
+     */
+
+    simpleJavaScriptCall(): void {
```

Comment:
I made a point about not touching _any_ of this code, partly because it's too hard to tell the difference with this change of indentation level.